asynchat/pyproject.toml

```diff
@@ -24,5 +24,5 @@ find = {include = ["asynchat*"]}
 "Homepage" = "https://github.com/youknowone/python-deadlib"
 
 [build-system]
-requires = ["setuptools>=61.0"]
+requires = ["setuptools>=75.0"]
```

Comment:
Is this also intended to be upgraded? I'd like to know the reason if 61.0 doesn't fit